models/models.go

```diff
@@ -83,3 +84,11 @@ type SearchInput struct {
 	EndTime   time.Time `gorm:"type:datetime"`
 	GpsLocation
 }
+
+type Booking struct{
+	DBModel
+	Booker uint
```

Comment:
What does `Booker` field hold. Issn't it same as `OwnerID`. The naming is ambiguous. Please change it